lora/utils.py

```diff
@@ -112,15 +112,27 @@ def make_shards(weights: dict, max_file_size_gibibyte: int = 15):
     return shards
 
 
-def save_model(save_dir: str, weights, tokenizer, config):
+def save_model(save_dir: str, weights, tokenizer, config, de_quantize: bool = False):
     save_dir = Path(save_dir)
     save_dir.mkdir(parents=True, exist_ok=True)
-    shards = make_shards(weights)
+
+    shards = make_shards(weights, max_file_size_gibibyte=5)
+    shards_count = len(shards)
+    shard_file_format = (
+        "model-{:05d}-of-{:05d}.safetensors"
```

Comment:
It was a bad idea to not use this format in `mlx-lm` as well. We should update it there as well

Reply:
I updated it in lora PR https://github.com/ml-explore/mlx-examples/pull/337